kmymoneysettings.h is generated by the kmm_settings target. We need to ensure
that it is built before kmm_ofximport.

kmymoney/plugins/ofximport/dialogs/kofxdirectconnectdlg.cpp:19:10: fatal error: 'kmymoneysettings.h' file not found

--- kmymoney/plugins/ofximport/CMakeLists.txt.orig	2018-06-05 11:19:49 UTC
+++ kmymoney/plugins/ofximport/CMakeLists.txt
@@ -28,6 +28,8 @@ target_link_libraries(kmm_ofximport
                       ${LIBALKIMIA_LIBRARIES}
                       ${LIBOFX_LIBRARIES})
 
+add_dependencies(kmm_ofximport kmm_settings)
+
 ########### install files ###############
 
 install(FILES kmm_ofximport.rc
